src/main/java/duke/Task.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+    }
+
+    public String getDescription() {
+        return this.description;
+    }
+
+    public void markAsDone() {
```

Comment:
Careful about this part as coding standards state that setter methods for boolean should be in the form of "setDone0()". Otherwise overall good job on getters and setters for the Tasks classes.